lib/network_utils.ml

```diff
@@ -39,7 +39,15 @@ let inject_igmp_query_script = ref "/usr/libexec/xenopsd/igmp_query_injector.py"
 let mac_table_size = ref 10000
 let igmp_query_maxresp_time = ref "5000"
 
+let in_unit_test = ref false
```

Comment:
Is this reference manipulated from outside? Where? If not, this does not have to be a reference.